config/main_routes.rb

```diff
@@ -110,3 +123,82 @@ def add_routes(routes_name)
+  get 'choose_a_certified_company/:company', to: 'choose_a_certified_company_loa2#about', as: :choose_a_certified_company_about
+  post 'choose_a_certified_company', to: 'choose_a_certified_company_loa2#select_idp', as: :choose_a_certified_company_submit
+end
+
+# HUH-233: implement appropriate variant B routes
+constraints short_hub_v3_variant_b do
+  get 'about_certified_companies', to: 'about_loa2_variant_b#certified_companies', as: :about_certified_companies
+  get 'about_identity_accounts', to: 'about_loa2_variant_b#identity_accounts', as: :about_identity_accounts
+  get 'about_choosing_a_company', to: 'about_loa2_variant_b#choosing_a_company', as: :about_choosing_a_company
+
+  get 'will_it_work_for_me', to: 'will_it_work_for_me#index', as: :will_it_work_for_me
+  post 'will_it_work_for_me', to: 'will_it_work_for_me#will_it_work_for_me', as: :will_it_work_for_me_submit
+
+  get 'select_documents', to: 'select_documents#index', as: :select_documents
+  get 'select_documents_none', to: 'select_documents#no_documents', as: :select_documents_no_documents
+  post 'select_documents', to: 'select_documents#select_documents', as: :select_documents_submit
+
+  get 'select_phone', to: 'select_phone_variant_b#index', as: :select_phone
+  post 'select_phone', to: 'select_phone_variant_b#select_phone', as: :select_phone_submit
+
+  get 'choose_a_certified_company', to: 'choose_a_certified_company_loa2_variant_b#index', as: :choose_a_certified_company
+  get 'choose_a_certified_company/:company', to: 'choose_a_certified_company_loa2_variant_b#about', as: :choose_a_certified_company_about
+  post 'choose_a_certified_company', to: 'choose_a_certified_company_loa2_variant_b#select_idp', as: :choose_a_certified_company_submit
+end
+
+# TODO (HUH-234) implement appropriate variant C routes
+constraints short_hub_v3_variant_c do
+  get 'will_it_work_for_me', to: 'will_it_work_for_me#index', as: :will_it_work_for_me
+  post 'will_it_work_for_me', to: 'will_it_work_for_me#will_it_work_for_me', as: :will_it_work_for_me_submit
+
+  get 'select_documents', to: 'select_documents_variant_c#index', as: :select_documents
+
+  #TODO: fix these to go with the vC controller
+  get 'select_documents_none', to: 'select_documents_variant_c#no_documents', as: :select_documents_no_documents
+  post 'select_documents', to: 'select_documents_variant_c#select_documents', as: :select_documents_submit
+  get 'select_documents_advice', to: 'select_documents_variant_c#advice', as: :select_documents_advice
+
+  get 'choose_a_certified_company', to: 'choose_a_certified_company_loa2_variant_c#index', as: :choose_a_certified_company
+  get 'choose_a_certified_company/:company', to: 'choose_a_certified_company_loa2_variant_c#about', as: :choose_a_certified_company_about
+  post 'choose_a_certified_company', to: 'choose_a_certified_company_loa_variant_c2#select_idp', as: :choose_a_certified_company_submit
```

Comment:
```suggestion
  post 'choose_a_certified_company', to: 'choose_a_certified_company_loa2_variant_c#select_idp', as: :choose_a_certified_company_submit
```
the `2` is in a wrong place :D

Reply:
:-o I'd sweat this worked